README.md

```diff
@@ -7,6 +7,19 @@ MyProxy is an application that proxies requests to other servers
 
 - [x] A valid Domain name `Eg: mydomain.com` (Currently supported providers: GoDaddy)
 
+- [x] Create an ssh server
+
+## Prerequisite examples
+
+Create ssh server: 
+
+DigitalOcean Droplets: [create my\_server\_ip here](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/)
+
+Sign up to GoDaddy and buy a domain:
```

Comment:
you can use the link here, instead of code blocks. https://www.markdownguide.org/basic-syntax